src/pages/AdminDashboard/CreateRoomForm.tsx

```diff
@@ -224,7 +252,9 @@ const CreateRoomForm = ({ fields, onFormSubmit, onCreateFormReset }) => {
                     id: fields.id,
                     destinations,
                 }
-                onFormSubmit(room, isEditing)
+
+                const remindAt = getRemindAt(room.startAt / 1000, notification)
```

Comment:
suggestion: for better readability use a var `const startAtInMillisecond = room.startAt / 1000`